src/util/scroll_to_first_error.ts

```diff
@@ -1,3 +1,7 @@
 export default function scrollToFirstError() {
-	document.getElementsByClassName( 'is-danger' )[ 0 ].scrollIntoView( { behavior: 'smooth', block: 'center', inline: 'nearest' } );
+	const errorElement = document.querySelector( '.help.is-danger' );
```

Comment:
Not a change request, but some small comment on performance, just for education: `getElementsByClassName` is twice as fast as `querySelector`. Probably because the former has a hash table, whereas the latter is more of a search operation (because it can process arbitrary CSS selectors). You can see for yourself at https://measurethat.net/Benchmarks/Show/5353/0/queryselector-vs-queryselectorall-vs-getelementsbyclass

See https://dev.to/wlytle/performance-tradeoffs-of-queryselector-and-queryselectorall-1074 for the rare exception (accessing/iterating over multiple elements, where the access time for individual elements is faster with querySelector).

In this case, it doesn't matter because the operation is performed only once and not millions of times. 

And of course, `querySelector` allows you query multiple classes on one element. I'm not sure if that benefits us here, but I also want this change to be merged so we'll just leave it.